lib/time.py

```diff
@@ -23,3 +62,67 @@ def sleep(secs):
 
 def time():
   return float(Now().UnixNano()) / Second
+
+
+_strftime_directive_map = {
+    '%': '%',
+    '0': NotImplementedError,
```

Comment:
Right I think we should leave out platform specific extensions like padding. From my previous comment:

"Thinking about this some more, I'm inclined to make this as simple as possible while still adhering to the Python spec. I don't see any value in supporting any extensions specific to one particular platform."